modules/ec2/main.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_instance" "Dev" {
   instance_type               = "t3.micro"
   subnet_id                   = var.subnet_ids[0]
   security_groups             = [aws_security_group.main.id]
-  key_name                    = "MyKeyPair"
+  #key_name                    = "MyKeyPair"
```

Comment:
Having commented-out code is a bad practice. When you remove something, just remove it.
Don't worry about having to revert things - the history will be still tracked in git
```suggestion
```

Reply:
resolved

Reply:
Still not fixed